crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -97,7 +97,7 @@ def finished?
 
     def save
       progress_file_path.open("w") do |f|
-        f.write(JSON.pretty_generate(progress))
+        f.write(YAML.dump(progress))
```

Comment:
still not convinced why to use yaml for the state, at least not from the commit message. isn't readability a better reason to use yaml, not some internal restrictions with symbols?
and not that it matters for much for crowbar, but when we have to read the file from our automation (bash) wouldn't json be better?